Sample/Sample/HostViewController.swift

```diff
@@ -35,13 +35,18 @@ class HostViewController: MenuContainerViewController {
         selectContentViewController(contentViewControllers.first!)
     }
     
-    override func menuTransitionOptionsBuilder() -> TransitionOptionsBuilder? {
-        return TransitionOptionsBuilder() { builder in
-            builder.duration = 0.6
-            builder.contentScale = 0.9
-        }
+    override func menuTransitionOptions() -> TransitionOptions? {
+        return TransitionOptions(duration: 0.4, contentScale: 0.9)
```

Comment:
Do we really need this callback if we already have `updateMenuTransition(options:)` method for changing transition options?